server/helpers/datahandlers.js

```diff
@@ -0,0 +1,36 @@
+module.exports.calcReviewsAverageStars = (averageStarsArray) => {
```

Comment:
I'm not sure how much of a difference it would make but is there a reason you didn't use mySQL built in functions to get the averages from your table.

Reply:
Good question.  My thinking - which is not something I'm 100% on, just a feeling - is that I still could, in some theoretical future, want a function that can pull the raw star review data, not just the averages.  So I decided to leave the query as just getting the raw data, then design the helper function separately.  That said, to your point the better thing to do probably would've been to an additional query that calc'd the averages for me (instead of using a separate helper).  You make a good point and I may re-factor depending on time.